src/Command/RectorCommand.php

```diff
@@ -93,39 +94,28 @@ protected function runRector(ConsoleIo $io, Arguments $args, string $autoload):
         );
         $io->verbose("Running <info>{$command}</info>");
 
-        $descriptorSpec = [
-            0 => ['pipe', 'r'],
-            1 => ['pipe', 'w'],
-            2 => ['pipe', 'w'],
-        ];
-        $process = proc_open(
-            $command,
-            $descriptorSpec,
-            $pipes,
-        );
-        if (!is_resource($process)) {
-            $io->error('Could not create rector process');
+        $io->info('Starting rector at ' . date('Y-m-d H:i:s'));
 
-            return false;
-        }
+        $process = Process::fromShellCommandline($command);
+        $process->setEnv($_ENV);
+        $process->setTimeout(null);
```

Comment:
The symfony process utility has a default 60 second timeout for commands it runs. Since it can take way longer for rector to finish this is the best way to go.